components/state/state.go

```diff
@@ -115,73 +115,73 @@ INSERT OR REPLACE INTO %s (%s, %s) VALUES (?, ?);
 
 func GetLoginInfo(ctx context.Context, dbRO *sql.DB, machineID string) (string, error) {
 	query := fmt.Sprintf(`
-SELECT %s FROM %s WHERE %s = '%s'
+SELECT COALESCE(%s, '') FROM %s WHERE %s = ?
```

Comment:
why?

Reply:
Removed, no need to handle this this way